data/tribes/scripting/help/controls.lua

```diff
@@ -39,117 +39,117 @@ local r = {
-               dl(help_format_hotkey(pgettext("hotkey", "Home")), _"Center main mapview on starting location") ..
+               dl(help_format_hotkey(wl.ui.get_shortcut("game_hq")), _"Center main mapview on starting location") ..
                -- TRANSLATORS: This is an access key combination. Localize, but do not change the key.
-               dl(help_format_hotkey(pgettext("hotkey", "Ctrl + 1-9")), _"Remember current location") ..
-               dl(help_format_hotkey(pgettext("hotkey", "1-9")), _"Go to previously remembered location") ..
+               dl(help_format_hotkey(help_set_landmarks_hotkeys()), _"Remember current location") ..
+               dl(help_format_hotkey(help_goto_landmarks_hotkeys()), _"Go to previously remembered location") ..
                -- TRANSLATORS: This is an access key combination. Localize, but do not change the key.
-               dl(help_format_hotkey(pgettext("hotkey", ", (comma)")), _"Go to previous location") ..
+               dl(help_format_hotkey(wl.ui.get_shortcut("quicknav_prev")), _"Go to previous location") ..
                -- TRANSLATORS: This is an access key combination. Localize, but do not change the key.
-               dl(help_format_hotkey(pgettext("hotkey", ". (period)")), _"Go to next location") ..
+               dl(help_format_hotkey(wl.ui.get_shortcut("quicknav_next")), _"Go to next location") ..
                -- TRANSLATORS: This is an access key combination. Localize, but do not change the key.
-               dl(help_format_hotkey(pgettext("hotkey", "Ctrl + +")), _"Increase zoom") ..
+               dl(help_format_hotkey(wl.ui.get_shortcut("zoom_in")), _"Increase zoom") ..
                -- TRANSLATORS: This is an access key combination. Localize, but do not change the key.
-               dl(help_format_hotkey(pgettext("hotkey", "Ctrl + -")), _"Decrease zoom") ..
+               dl(help_format_hotkey(wl.ui.get_shortcut("zoom_out")), _"Decrease zoom") ..
                -- TRANSLATORS: This is an access key combination. Localize, but do not change the key.
-               dl(help_format_hotkey(pgettext("hotkey", "Ctrl + 0")), _"Reset zoom") ..
+               dl(help_format_hotkey(wl.ui.get_shortcut("zoom_reset")), _"Reset zoom") ..
                -- TRANSLATORS: This is an access key combination. Localize, but do not change the key.
-               dl(help_format_hotkey(pgettext("hotkey", "Ctrl + F11")), _"Take a screenshot") ..
+               dl(help_format_hotkey(wl.ui.get_shortcut("screenshot")), _"Take a screenshot") ..
                toggle_fullscreen_hotkey ..
                -- TRANSLATORS: This is an access key combination.
-               dl(help_format_hotkey(pgettext("hotkey", "F6")), _"Show the debug console (only in debug-builds)")
+               dl(help_format_hotkey(pgettext("hotkey", "Ctrl + Shift + Space")), _"Show the debug console (only in debug-builds)")
```

Comment:
Must check this on macOS , F6 would be ok but this is +/- resverd for some system function :-/

Reply:
The reason why I changed this away from F6 to now Ctrl-Shift-Space is that some users might want to assign F1-F12 for stuff like stock menus or landmarks, so a hardcoded shortcut on F6 would get in the way